src/flags.go

```diff
@@ -174,6 +175,12 @@ func propagateFlags() {
 		PanicWithMessage("Non-empty data-body was provided, but no request type was given. Hence, encoding of data-body is not possible.")
 	}
 
+	if strings.HasPrefix(strings.TrimSpace(CurrentConfig.DataText), "@") {
```

Comment:
Why is the `strings.TrimSpace` used here? The shell itself should take care of removing unnecessary spaces e.g. `-d    @payload.txt   <next arg>`. And if the filename itself contains intentional spaces, then we better use it. e.g. `-d "@pay load. txt  "`

Reply:
`strings.TrimSpace` is not needed for the e.g. u mentioned `-d "@pay load. txt  "`. Thanks.